src/Components/Cart/Cartproduct.js

```diff
@@ -1,9 +1,13 @@
-import { Link } from "react-router-dom";
 import { useCart } from "../../Context/DataContext/CartContext";
+import { useWishlist } from "../../Context/DataContext/WishlistContext";
 export const CartProduct = ({product}) =>{
 const {_id, name, image, description, price , discount} = product
```

Comment:
Use a formatter like Prettier to format your code. This improves code readability.